snapcraft/main.py

```diff
@@ -308,7 +308,13 @@ def run(args, project_options):  # noqa
         parts.define(args['<part-name>'])
     elif args['search']:
         parts.search(' '.join(args['<query>']))
+    elif os.environ.get('SNAPCRAFT_CONTAINER_BUILDS'):
```

Comment:
Right now only "snapcraft"/ "snapcraft snap" honors the flag. There's a separate task "pass through snapcraft commands into the container" in the backlog which is about that (unless I misunderstood). So I was assuming once this branch is done, we would discuss and integrate the other commands.